torch_xla/csrc/tensor.h

```diff
@@ -189,8 +189,9 @@ class XLATensor {
 
   static XLATensor t(const XLATensor& input);
 
-  XLATensor view(
-      tensorflow::gtl::ArraySlice<const xla::int64> output_size) const;
+  static XLATensor view(
```

Comment:
I don't mind doing this for consistency, but there was a reason for keeping this non-static: there's no `at::view` wrapper, it's one of the few methods which looks like that.